data/json/monsters/mammal.json

```
@@ -2521,7 +2521,7 @@
   {
     "id": "mon_reindeer",
     "type": "MONSTER",
-    "name": { "str": "Reindeer" },
+    "name": { "str": "reindeer" },
```

```suggestion
    "name": { "str_sp": "reindeer" },
```
https://en.wiktionary.org/wiki/reindeer says `reindeers` is only used in biology to refer to multiple species.